framework/src/userobject/SolutionUserObject.C

```diff
@@ -363,10 +373,9 @@ SolutionUserObject::readExodusII()
     for (const auto & var_name : _elemental_variables)
       _exodusII_io->copy_elemental_solution(*_system, var_name, var_name, _exodus_time_index);
 
-    if (_scalar_variables.size() > 0)
+    if (!_scalar_variables.empty())
```

Comment:
please revert that it s the same

Reply:
This was suggested by cody and derek haha, look at the resolved comments

Reply:
lol

Reply:
ok I m fine with it and they are at fault